lib/elixir/lib/system.ex

```diff
@@ -471,6 +474,35 @@ defmodule System do
     :erlang.halt(String.to_charlist(status))
   end
 
+  @doc """
+  Stops the Erlang runtime system.
+
+  Stops the Erlang runtime system carefully. All applications are taken down
```

Comment:
I think first sentence of this paragraph could serve as function summary as it worth to mention "carefully" there anyway, and not to duplicate information.